public/app/features/integrated-alerting/components/NotificationChannel/NotificationChannel.service.ts

```diff
@@ -39,6 +40,10 @@ const TO_MODEL = {
 };
 
 const getType = (channel: NotificationChannelResponse): NotificationChannelType => {
+  if ('email_config' in channel) {
```

Comment:
nit: We are not leveraging typescript here! 
eslint could help via https://eslint.org/docs/rules/no-restricted-syntax, but this is for another PR I guess

Reply:
Done!